docs/admin/multi-cluster.md

```diff
@@ -80,8 +80,9 @@ Second, decide how many clusters should be able to be unavailable at the same ti
 the number that can be unavailable `U`.  If you are not sure, then 1 is a fine choice.
 
 If it is allowable for load-balancing to direct traffic to any region in the event of a cluster failure, then
-you need `R + U` clusters.  If it is not (e.g you want to ensure low latency for all users in the event of a
-cluster failure), then you need to have `R * U` clusters (`U` in each of `R` regions).  In any case, try to put each cluster in a different zone.
+you need at least `min(R, U+1)` clusters.  If it is not (e.g you want to ensure low latency for all users
```

Comment:
As far as I can determine, this is still not quite right?  Or at least not clearly articulated.  Here's a strawman proposal for your consideration.  Feel free to reword it more simply if necessary - I've been overly verbose to ensure that you understand my confusion/proposed clarification:

1. Assume that the steady state (no failure) configuration is to have sufficient capacity in one cluster in each region to serve the traffic originating in that region from that region.  Hence 'R' clusters.
2. If cross-region load balancing is permissable, and 'U' cluster failure needs to be handled gracefully, add an additional 'U' clusters globally (hence 'R + U' in total).
3. If cross-region load balancing is not permissable, add an additional 'U' clusters per region (i.e. R + UR  = R * (U + 1) in total).